### PR TITLE
machines: Allow a space characters in arguments of virt-install

### LINKED ISSUE #9623: machines: escape arguments of virsh
Escape shell script command arguments when calling `virsh` from `pkg/machines`.

VM names, storage pool names and others can consist of all characters as file-names do, means i.e. quotes, spaces, etc.

Related to `pkg/machines/libvirt.es6`.

Issue identified within #9166 .

### LINKED ISSUE #14524: Virtual machines: problem with filenames containing spaces
Cockpit version: 226
OS: Fedora 32
Page: Virtual Machines

The creation of a new VM fails, when the filename of an ISO image contains spaces (e.g. "Windows Server X64 2019.iso"). This will lead to the following error message:

`Creation of VM failed: usage: virt-install --name NAME --memory MB STORAGE INSTALL [options] virt-install: error: unrecognized arguments: Server X64 2019.iso`

The above filenames should probably be in quoted/escaped before the command is executed.

### PR DESCRIPTION
**Pull request by skobyda**
Closes #14524
Closes #9623

### CODE REVIEW

File: pkg/machines/scripts/install_machine.sh [hunk 1]
```diff
@@ -105,22 +105,22 @@ handleFailure() {
...
-    $BOOT_PARAM \
-    $AUTOSTART_PARAM || handleFailure $?
+    "$MEMORY_PARAM" \
+    "$DISKS_PARAM" \
+    "$INSTALL_METHOD" \
+    "$GRAPHICS_PARAM" \
+    "$VNICS_PARAM" \
+    "$VCPUS_PARAM" \
+    "$BOOT_PARAM" \
+    "$AUTOSTART_PARAM "|| handleFailure $?
```
Comment (KKoukiou) [inline pkg/machines/scripts/install_machine.sh:123]: Wrong spacing here after parameter, not that it matters but for consistency.

**Review by KKoukiou — CHANGES_REQUESTED** [review]
This rather needs a test case with a  disk && vm name with spaces in the name.

**Review by KKoukiou — COMMENTED** [review]
@skobyda this is only missing a test, otherwise I am happy with it

**Comment by skobyda** [comment]
I added a test, it passes, and all the failures are flakes which I have seen in other cases. @KKoukiou can you give it another look?

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**